protocol/x/clob/types/keys.go

```diff
@@ -18,94 +18,91 @@ const (
-	OrdersFilledDuringLatestBlockKey = "orders_filled_during_latest_block"
+	BlockHeightToPotentiallyPrunableOrdersPrefix = "HgtToPrunable/"
 )
 
 // Below key prefixes are not explicitly used to read/write to state, but rather used to iterate over
 // certain groups of items stored in state.
 const (
 	// StatefulOrderKeyPrefix is the prefix key for all long term orders and all conditional orders,
 	// both triggered and untriggered.
-	StatefulOrderKeyPrefix = "stateful_order_placement/"
+	StatefulOrderKeyPrefix = "SOP/"
 
 	// PlacedStatefulOrderKeyPrefix is the prefix key for placed long term orders and triggered
 	// conditional orders. It represents all stateful orders that should be placed upon the memclob
 	// during app start up.
-	PlacedStatefulOrderKeyPrefix = StatefulOrderKeyPrefix + "placed/"
+	PlacedStatefulOrderKeyPrefix = StatefulOrderKeyPrefix + "P/"
 )
 
 // Store / Memstore
 const (
 	// TriggeredConditionalOrderKeyPrefix is the key to retrieve an triggered conditional order and
 	// information about when it was triggered.
-	TriggeredConditionalOrderKeyPrefix = PlacedStatefulOrderKeyPrefix + "conditional/"
+	TriggeredConditionalOrderKeyPrefix = PlacedStatefulOrderKeyPrefix + "C/"
 
 	// LongTermOrderPlacementKeyPrefix is the key to retrieve a long term order and information about
 	// when it was placed.
-	LongTermOrderPlacementKeyPrefix = PlacedStatefulOrderKeyPrefix + "long_term/"
+	LongTermOrderPlacementKeyPrefix = PlacedStatefulOrderKeyPrefix + "LT/"
 
 	// UntriggeredConditionalOrderKeyPrefix is the key to retrieve an untriggered conditional order and
 	// information about when it was placed.
-	UntriggeredConditionalOrderKeyPrefix = StatefulOrderKeyPrefix + "untriggered/conditional/"
+	UntriggeredConditionalOrderKeyPrefix = StatefulOrderKeyPrefix + "Untrig/"
 
 	// StatefulOrdersTimeSlicePrefix is the key to retrieve a unique list of the stateful orders that
 	// expire at a given timestamp, sorted by order ID.
-	StatefulOrdersTimeSlicePrefix = "stateful_orders_time_slice/"
+	StatefulOrdersTimeSlicePrefix = "ExpTime/"
 )
 
 // Transient Store
 const (
 	// NextStatefulOrderBlockTransactionIndexKey is the transient store key that stores the next
 	// transaction index to use for the next newly-placed stateful order.
-	NextStatefulOrderBlockTransactionIndexKey = "next_stateful_order_block_transaction_index"
+	NextStatefulOrderBlockTransactionIndexKey = "NextTxIdx"
 
 	// UncommittedStatefulOrderPlacementKeyPrefix is the key to retrieve an uncommitted stateful order and information
 	// about when it was placed. uncommitted orders are orders that this validator is aware of that have yet to be
 	// committed to a block and are stored in a transient store.
-	UncommittedStatefulOrderPlacementKeyPrefix = StatefulOrderKeyPrefix + "uncommitted/long_term/"
+	UncommittedStatefulOrderPlacementKeyPrefix = StatefulOrderKeyPrefix + "U/LT/"
```

Comment:
Why is `StatefulOrderKeyPrefix` used here? It seems unrelated. Originally this was just a string and I broke it out into explicitly having this prefix since the prefix did match and I thought it was needed for iterating over the keys, but I don't think it is

Reply:
not needed for iteration. 

I think having `SOP` here is fine since the mapped value is of type `LongTermOrderPlacement` which is the same as other keys that are prefixed by `SOP`.

No strong opinion though. let me know if you have a strong preference to make these different

Reply:
actually let me just update and remove SOP for simplicity.